backend/coins/btc/account.go

```diff
@@ -574,10 +574,12 @@ func (account *Account) Balance() (*accounts.Balance, error) {
 	if account.fatalError.Load() {
 		return nil, errp.New("can't call Balance() after a fatal error")
 	}
+	if !account.Synced() {
+		return nil, accounts.ErrSyncInProgress
+	}
 	balance, err := account.transactions.Balance()
 	if err != nil {
-		// TODO
-		panic(err)
+		return nil, err
```

Comment:
This should probably still panic or at least log the error, as this could only happen with something like a corrupt DB or IO errors.